lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorSettings.cs

```diff
@@ -89,7 +96,22 @@ internal sealed class EvaluatorSettings
                 Utilities.Diagnostics.Exceptions.Throw(
                     new ArgumentException("empty or null rootContextConfigString"), Logger);
             }
-            _rootContextConfig = new ContextConfiguration(rootContextConfigString);
+            _rootContextConfig = _serializer.FromString(rootContextConfigString);
+
+            try
+            {
+                _rootContextId = injector.ForkInjector(_rootContextConfig).GetNamedInstance<ContextConfigurationOptions.ContextIdentifier, string>();
```

Comment:
A best practice to get value from a namedParameter is to put it in a constructor and inject that object. That is the reason I feel we should still wrap Context configuration in a class, different from the one for configuration module, and have this class contain all the named parameters related to context configuration. I know for now, we only have identifier though.